### ISSUE TITLE
inprocess example

### ISSUE THREAD

**Issue by joeblew99** [mosuka/indigo#2]
this is a nice clean room example of using bleve.

Could you add an inprocess example please ?
At the moment you have a GRPC, REST setup, but i would also like to run this embedded on Desktops and mobiles and so a simple inprocess example will make it useful for me and others.

**Comment by mosuka** [comment]
Hi @joeblew99 ,
How about this?
https://github.com/mosuka/bleve-example

**Comment by gedw99** [comment]
nah, i think its MUCH better to use your indigo proto api as its really a perfect API from what i can see.
What about starting in the examples an embedded example that use the same text data your using.
that way in terms of CI / Testing both the Remote and Embedded examples use the same test data and stay in sync etc.

It also easy to use your nice proto to extend to use NATS and go-kit actually with not much work later too. But i can certainly understand if you dont want that too
